test/functional/test_1_browser_call.py

```diff
@@ -310,12 +312,33 @@ def local_check_connection_length_noted(self):
                            "OTSdkDriver._connectionLengthNotedCalls should be "
                            "> 0, noted_calls = " + str(noted_calls))
 
+    def local_close_conversation(self):
+        self.marionette.set_context("chrome")
+        self.marionette.switch_to_frame()
+
+        chatbox = self.wait_for_element_exists(By.TAG_NAME, 'chatbox')
+        close_button = chatbox.find_element("anon attribute", {"class": "chat-loop-hangup chat-toolbarbutton"})
```

Comment:
If you rebase onto latest master, you should be able to change `"anon attribute"` to `By.ANON_ATTRIBUTE`